.github/workflows/backend.yml

```
@@ -24,6 +24,8 @@ on:
       - dev
     paths-ignore:
       - 'docs/**'
+      - 'README.md'
+      - 'README_zh_CN.md'
```

Ignore all markdown files would be better
```suggestion
      - '**/*.md'
```

---

SGTM, addressed